catalog/nginx-web.conf

```diff
@@ -20,6 +20,10 @@ server {
         expires -1;
     }
 
+    location = /__embed {
```

Comment:
You should also add the debug harness, too - might be useful.

Reply:
should we expose it in production?

Reply:
done